starter-code/src/component/Search.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+
+import './Search.css';
+
+import 'bulma/css/bulma.css';
+
+class FoodResearch extends Component {
+    constructor() {
+      super()
+      this.state = {
+        FoodArray: [],
+        FilteredFood: []
+      }
+    }
+  
+    componentWillMount() {
+      this.setState({
+        FoodArray,
+        FilteredFood: FoodArray
+      })
+    }
+  
+    filterPoets = (foodFilter) => {
+      let FilteredFood = this.state.FoodArray
+      FilteredFood = FilteredFood.filter((food) => {
+        let foodName = food.firstName.toLowerCase() + food.lastName.toLowerCase()
+        return foodName.indexOf(
+          foodFilter.toLowerCase()) !== -1
+      })
+      this.setState({
+        FilteredFood
+      })
+    }
+  
+    render() {
+      return (
+        <Poets FoodArray={this.state.FilteredFood} match={this.props.match} onChange={this.filterPoets} />
```

Comment:
Where does this Poets come from? 🤷🏽‍♀️

Reply:
It can't work because there is no component called `Poets` and here you are calling it